lib/testing/src/lib/process-services/pages/start-process-dialog.page.ts

```diff
@@ -18,26 +18,27 @@
 import { BrowserVisibility } from '../../core/utils/browser-visibility';
 import { by, element } from 'protractor';
 import { BrowserActions } from '../../core/utils/browser-actions';
+import { ElementFinder } from 'protractor/built/element';
```

Comment:
correct import would be from 'protractor' , though I don't think this exact one causes harm